app/scripts/models/reliers/oauth.js

```diff
@@ -16,9 +16,49 @@ define(function (require, exports, module) {
   var RelierKeys = require('lib/relier-keys');
   var Url = require('lib/url');
   var Validate = require('lib/validate');
+  var Validator = require('vat');
 
   var RELIER_FIELDS_IN_RESUME_TOKEN = ['state', 'verificationRedirect'];
 
+  /*eslint-disable camelcase*/
+  var CLIENT_INFO_SCHEMA = {
+    id: Validator.string().required().use(Validate.isHexValid).as('clientId'),
+    image_uri: Validator.string().optional().allow('').use(Validate.isUrlValid).as('imageUri'),
```

Comment:
> `use` doesn't seem like the right verb...

To try and keep things suitable javascripty, how about `apply`? I know it isn't exactly the same concept as `Function.apply`, but it's kind of similar?